[2022-10-16 22:38:53,242] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dbt.dbt_deps scheduled__2022-10-14T00:00:00+00:00 [queued]>
[2022-10-16 22:38:53,249] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dbt.dbt_deps scheduled__2022-10-14T00:00:00+00:00 [queued]>
[2022-10-16 22:38:53,249] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-16 22:38:53,249] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-10-16 22:38:53,249] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-16 22:38:53,267] {taskinstance.py:1377} INFO - Executing <Task(DbtDepsOperator): dbt_deps> on 2022-10-14 00:00:00+00:00
[2022-10-16 22:38:53,274] {standard_task_runner.py:52} INFO - Started process 253288 to run task
[2022-10-16 22:38:53,280] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dbt', 'dbt_deps', 'scheduled__2022-10-14T00:00:00+00:00', '--job-id', '97', '--raw', '--subdir', 'DAGS_FOLDER/dbt_airflow.py', '--cfg-path', '/tmp/tmp5d7atnpr', '--error-file', '/tmp/tmp_mow1_k5']
[2022-10-16 22:38:53,281] {standard_task_runner.py:80} INFO - Job 97: Subtask dbt_deps
[2022-10-16 22:38:53,331] {task_command.py:370} INFO - Running <TaskInstance: dbt.dbt_deps scheduled__2022-10-14T00:00:00+00:00 [running]> on host ubuntu
[2022-10-16 22:38:53,382] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dbt
AIRFLOW_CTX_TASK_ID=dbt_deps
AIRFLOW_CTX_EXECUTION_DATE=2022-10-14T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-14T00:00:00+00:00
[2022-10-16 22:38:53,383] {dbt_hook.py:117} INFO - dbt deps
[2022-10-16 22:38:53,389] {dbt_hook.py:126} INFO - Output:
[2022-10-16 22:38:56,387] {dbt_hook.py:130} INFO - 05:38:56  Running with dbt=1.1.0
[2022-10-16 22:38:56,401] {dbt_hook.py:130} INFO - 05:38:56  Warning: No packages were found in packages.yml
[2022-10-16 22:38:57,719] {dbt_hook.py:132} INFO - Command exited with return code 0
[2022-10-16 22:38:57,731] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dbt, task_id=dbt_deps, execution_date=20221014T000000, start_date=20221017T053853, end_date=20221017T053857
[2022-10-16 22:38:57,781] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-16 22:38:57,816] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
